test/podman_networking.bats

```diff
@@ -21,3 +21,24 @@ function setup() {
     echo "$output"
     [ "$status" -eq 0 ]
 }
+
+@test "expose port 222" {
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} run -dt --expose 222-223 ${ALPINE} /bin/sh
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run bash -c "iptables -t nat -L"
```

Comment:
Why are you running iptables twice?

Reply:
ideally, i want to check for that both 222 and 223 are present to be sure ranges work.  right now, this test is ripped apart to figure out why it is failing on ubuntu.